roles/openshift_cluster_monitoring_operator/files/cluster-monitoring-operator.yaml

```diff
@@ -0,0 +1,39 @@
+---
+apiVersion: extensions/v1beta1
```

Comment:
I think this needs converted into a template.openshift.io/v1.Template like the [web console](https://github.com/openshift/openshift-ansible/blob/master/roles/openshift_web_console/files/console-template.yaml)